umh-core/pkg/config/config.go

```diff
@@ -32,13 +33,14 @@ type FullConfig struct {
 }
 
 type InternalConfig struct {
-	Services        []S6FSMConfig           `yaml:"services,omitempty"`        // Services to manage, can be updated while running
-	Benthos         []BenthosConfig         `yaml:"benthos,omitempty"`         // Benthos services to manage, can be updated while running
-	Nmap            []NmapConfig            `yaml:"nmap,omitempty"`            // Nmap services to manage, can be updated while running
-	Redpanda        RedpandaConfig          `yaml:"redpanda,omitempty"`        // Redpanda config, can be updated while running
-	BenthosMonitor  []BenthosMonitorConfig  `yaml:"benthosMonitor,omitempty"`  // BenthosMonitor config, can be updated while running
-	Connection      []ConnectionConfig      `yaml:"connection,omitempty"`      // Connection services to manage, can be updated while running
-	RedpandaMonitor []RedpandaMonitorConfig `yaml:"redpandaMonitor,omitempty"` // RedpandaMonitor config, can be updated while running
+	Services          []S6FSMConfig             `yaml:"services,omitempty"`          // Services to manage, can be updated while running
```

Comment:
Just as a note: We should keep one standard of indentation/formatting.
I would just go for https://pkg.go.dev/cmd/gofmt without any special rules.

Reply:
it was reformatted because ProtocolConverterConfig is logner than the rest so all the `yaml:...` were aligned, this is default behaviour